oat-sa/tao-test-runner-qti-fe
# Fix/tr 199 flat radio navigation

Related to https://oat-sa.atlassian.net/browse/TR-199

- enable flat navigation for radio inputs on NFER tests with REBL theme
- load modules with correct namespace to avoid double bundling

Required: 
- https://github.com/oat-sa/extension-tao-testqti/pull/1980
- https://github.com/oat-sa/extension-tao-nfer/pull/254

How to check:
 - create delivery with more than one test item and REBL theme enabled
 - check the inputs are accesible via tab key
